packages/connectors/src/MediaConnector.ts

```diff
@@ -43,4 +71,5 @@ export type MediaConnectorCapabilities = {
     detail: boolean;
     filtering: boolean;
     metadata?: boolean;
+    analyzeSmartCrop?: boolean;
```

Comment:
I don't understand why we need to add `smart crop` in the capabilities, considering `analyze` method also doesn't specify it. 

```suggestion
    analyze?: boolean;
```

Reply:
Because we will have more things to come in analyze method later

Reply:
Yeah after checking the engine implementation I now understand _why_. 

However this is not really in line with other capability names, was this done on purpose? 

Considering `filtering` is also not called `queryFiltering` and we make a split between `query` and `filtering` capability.

```suggestion
    analyze?: boolean; // the analyze method exists
    smartCrop?: boolean; // smart crop feature is available in analyze
```